articles/005-fee-elimination.md

```diff
@@ -0,0 +1,125 @@
+# Fee Elimination on Casper 2.0
+
+The Casper 2.0 (aka Condor) network upgrade introduces new options to the way a Casper Network can be configured to handle gas consumption. Scheduled along with the release of Condor into Mainnet is a change in the configuration of the Casper Network to use a model known as **Fee Elimination** for gas payments. The purpose of this article is to introduce this model, and describe how Fee Elimination will affect the behaviour of the Casper Network once Condor is released.
```

Comment:
"Behaviour" : we should probably decide to use American English across the board right? My bad.